### DISCUSSION TITLE
Is Office URI Schemes supported

### DISCUSSION THREAD

**Discussion by tbergholt** [microsoft-search/pnp-modern-search#3595] [Q&A] [answered]
Trying to use Office URI Schemes in HTML link in the Results web part, but can not getting it to work and do not know about Office URI Schemes is supported.
An example could be like, while trying to open a template as a new document (it should not be necessary to add the save location to the  syntax)
```
{{else contains (slot item @root.slots.FileType) "xltx"}}
<a href="ms-excel:nft|u|{{slot item @root.slots.Path}}" target="_blank" style="color: && {{@root.theme.semanticColors.link}}">
	{{slot item @root.slots.Title}}
</a>
```

**Comment by till-llit** [comment, marked as answer]
until 4.9.0 they where supported.

from 4.9.0 until now they are filtered by url sanitzier.

with this pull request they should work again (hopefully in the next version):

https://github.com/microsoft-search/pnp-modern-search/pull/3584

**Reply by tbergholt** [reply]
Thanks for quick response ...

**Reply by till-llit** [reply]
with 4.11.0 it should work again, you can find it here:

https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.11.0

**Comment by tbergholt** [comment]
Thanks ... So all the URI Schemes are working again?
________________________________

**Reply by till-llit** [reply]
this is the current regex to validate the URI scheme:

`ALLOWED_URI_REGEXP = '/^(?:(?:(?:f|ht)tps?|mailto|file|tel|callto|im|cid|xmpp|xxx|ms-\w+):|[^a-z]|[a-z+.\-]+(?:[^a-z+.\-:]|$))/i';`

i just added another PR to ad "msteams" to the list.

you can validate your urls with with regex to see what will happen.
